pipelines/nextflow/modules/process_gff3.nf

```diff
@@ -20,4 +20,26 @@ process PROCESS_GFF3 {
     """
     process_gff3 --in_gff_path ${gff3} --genome_data ${genome}
     """
-}
```

Comment:
Can we keep these two processes separate ? Better to keep one process per module file IMO

Reply:
as it stands : process_gff3.nf & gff3_validation.nf

Reply:
Yes, we need to do this for several of those. I'll do that now

Reply:
Done

Reply:
Brill thanks Mat !